src/test/java/de/tum/in/www1/artemis/service/ExerciseDateServiceTest.java

```diff
@@ -147,6 +147,33 @@ void itShouldAlwaysBeBeforeANonExistingDueDate() {
         assertThat(exerciseDateService.isAfterDueDate(participation)).isFalse();
     }
 
+    @Test
```

Comment:
Optional: You can think about using a parameterized test here

Reply:
I don't think this would simplify the code here, also all tests in this file a currently written in a consistent form.